docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -464,6 +464,31 @@ Only explict way of setting the hostname is by setting `eureka.instance.hostname
 You can set your hostname at the run-time by using an environment variable -- for example, `eureka.instance.hostname=${HOST_NAME}`.
 ====
 
+=== Securing The Eureka Server
+
+You can secure your Eureka server simply by adding Spring Security to your
+server's classpath via `spring-boot-starter-security`.  By default when Spring Security is on the classpath it will require that
+a valid CSRF token be sent with every request to the app.  Since Eureka clients will not
+generate a valid cross site request forgery (CSRF) token you will need to disable this requirement for the `/eureka/**` endpoints.
```

Comment:
No client generates a CSRF token. Maybe rephrase this as "Eureka clients will not generally possess a valid..."?